converters/fromZigbee.js

```diff
@@ -3075,7 +3075,7 @@ const converters = {
             case tuya.dataPoints.moesSwindowDetectionFunktion_A2:
                 return {window_detection: value ? 'ON' : 'OFF'};
             case tuya.dataPoints.moesSwindowDetection:
-                return {window_detection: value ? 'ON' : 'OFF'};
+                return {window: value ? 'CLOSED' : 'OPEN'};
```

Comment:
I don't understand why this has been renamed, could you explain it?

Reply:
These are two different functions.
1. Window detection can be switched on and off.
2. Shows whether Window is recognized as opened or closed. For control.

![Bildschirmfoto vom 2021-06-14 18-22-44](https://user-images.githubusercontent.com/45510877/121928564-8ea96800-cd40-11eb-9115-fa6ad95575da.png)
![Bildschirmfoto vom 2021-06-14 18-18-25](https://user-images.githubusercontent.com/45510877/121928590-9537df80-cd40-11eb-90e4-60f7129dd0e1.png)